content/en/blog/_posts/2020-08-26-generic-ephemeral-volumes.md

```diff
@@ -0,0 +1,393 @@
+---
+title: Ephemeral volumes with storage capacity tracking: EmptyDir on steroids
+date: 2020-08-26
```

Comment:
layout: blog
title: 'Ephemeral volumes with storage capacity tracking: EmptyDir on steroids'
date: 2020-09-01
slug: ephemeral-volumes-with-storage-capacity-tracking

Reply:
I added that header and renamed the file to match that date.